tutorial/20_recipes/008_specify_params.py

```diff
@@ -0,0 +1,121 @@
+"""
+.. _specify_params:
+
+Specify Hyperparameters
+=======================
+
+Sometimes, it is natural for you to try some experiments with your out-of-box hyperparameters.
+For example, you have some specific sets of hyperparameters to try in your mind before using Optuna for the best hyperparameters.
+
+First scenario
+--------------
+
+Try some sets of hyperparameters with :func:`optuna.study.Study.enqueue_trial`.
+"""
+
+import lightgbm as lgb
+import numpy as np
+import sklearn.datasets
+import sklearn.metrics
+from sklearn.model_selection import train_test_split
+
+import optuna
+
+
+###################################################################################################
+# Define the objective function.
+def objective(trial):
+    data, target = sklearn.datasets.load_breast_cancer(return_X_y=True)
+    train_x, valid_x, train_y, valid_y = train_test_split(data, target, test_size=0.25)
+    dtrain = lgb.Dataset(train_x, label=train_y)
+    dvalid = lgb.Dataset(valid_x, label=valid_y)
+
+    param = {
+        "objective": "binary",
+        "metric": "auc",
+        "verbosity": -1,
+        "boosting_type": "gbdt",
+        "bagging_fraction": trial.suggest_float("bagging_fraction", 0.4, 1.0 + 1e-12),
```

Comment:
A nit, but the `bagging_fraction` should be equal to or less than 1 (c.f., [LightGBM reference](https://lightgbm.readthedocs.io/en/latest/Parameters.html#bagging_fraction)).

```suggestion
        "bagging_fraction": min(trial.suggest_float("bagging_fraction", 0.4, 1.0 + 1e-12), 1),
```